broker/src/main/java/org/apache/rocketmq/broker/BrokerStartup.java

```diff
@@ -187,6 +187,13 @@ public static BrokerController createBrokerController(String[] args) {
             JoranConfigurator configurator = new JoranConfigurator();
             configurator.setContext(lc);
             lc.reset();
+            System.setProperty("brokerLogDir", "");
```

Comment:
If it is set to "" here, will it be in the original directory by default?

Reply:
yes, if brokerLogDir is "", the logPath in logback_broker.xml  is default.